runtime/asgard/src/lib.rs

```diff
@@ -1409,6 +1411,17 @@ impl bifrost_vsbond_auction::Config for Runtime {
 	type WeightInfo = weights::bifrost_vsbond_auction::WeightInfo<Runtime>;
 }
 
+impl merkle_distributor::Config for Runtime{
+	type Event = Event;
+	type CurrencyId = CurrencyId;
+	type MultiCurrency = Currencies;
+	type Balance = u128;
```

Comment:
Balance type should be defined align with Bifrost balnace type. In case the chain balance type changes, this module balance can change as well. 
`type Balance = Balance;`

Reply:
Yes, it should be.

Reply:
It has been updated